server/src/services/desks.service.ts

```diff
@@ -1,13 +1,19 @@
-import { DeskDto } from "../models/desk.model";
+import { DeskDto, DeskStatus } from "../models/desk.model";
 import fs from "fs";
 import * as path from "path";
 import { parse } from "csv-parse";
+import { DataPoint } from "../client/influx.client";
+import {
+  getLatestActiveDataPoint,
+  getLatestDataPoint,
+  getStats,
+} from "./influx.service";
 
 type GetDesksServiceParams = {
   zoneId: string;
   deskIds?: string[];
   count?: number;
-  unit?: "day" | "week" | "month" | "year";
+  unit?: "days" | "weeks" | "months" | "years";
```

Comment:
proc si tam pridaval ty eska?:D Bez nich je to lepsi

Reply:
Ok, opravim to i ve swaggeru